packages/fxa-settings/src/components/UnitRowSecondaryEmail/index.tsx

```diff
@@ -110,38 +234,42 @@ export const UnitRowSecondaryEmail = () => {
           )}
         </div>
         <div className="unit-row-actions" data-testid="unit-row-actions">
-          {verified && (
-            <div>
+          <div className="flex items-center -mt-1">
+            {verified && (
               <button
-                className="cta-neutral transition-standard"
+                className="cta-neutral cta-base transition-standard"
                 onClick={() => {
-                  // Make secondaryEmail the primary email!
+                  queueEmailAction(makeEmailPrimary);
```

Comment:
We should probably add a loading state for clicking "Make Primary" and "delete" to show the loading spinner. I can double click and cause an error message to show pretty easily.

Reply:
Oh good catch.

Reply:
We have some work to do with LoadingSpinner, it doesn't quite work as I'd hoped, and I want to avoid making this PR larger than it already is, so for now I've set the buttons disabled (visibly and functionally) when they're clicked.

Reply:
Oh bah, there's already an issue related to this 🙃  https://jira.mozilla.com/browse/FXA-2152